start.py

```diff
@@ -26,6 +26,9 @@
 
 
 def main(args):
+    if args.license_path and not args.license_credentials:
+        raise Exception('License-credentials is required argument if license-path is provided')
```

Comment:
=> `raise Exception('--license-credentials is a required argument if --license-url is provided.')`